sdk/textanalytics/Azure.AI.TextAnalytics/samples/Sample10_MultiLabelClassify.md

```diff
@@ -14,9 +14,9 @@ TextAnalyticsClient client = new(endpoint, credential);
 
 The values of the `endpoint` and `apiKey` variables can be retrieved from environment variables, configuration settings, or any other secure approach that works for your application.
 
-## Performing Custom Multiple Label Classification in one or multiple documents
+## Perform custom multi-label classification on one or more text documents
 
-To perform Multiple Label Classification in one or multiple documents, set up a `MultiLabelClassifyAction` and call `StartAnalyzeActionsAsync` on the documents. The result is a Long Running Operation of type `AnalyzeActionsOperation` which polls for the results from the API.
+To perform custom multi-label classification one or more text documents, call `MultiLabelClassifyAsync` on the `TextAnalyticsClient` by passing the documents as either an `IEnumerable<string>` parameter or an `IEnumerable<TextDocumentInput>` parameter. This returns an `ClassifyDocumentOperation`.
```

Comment:
```suggestion
To perform custom multi-label classification one or more text documents, call `MultiLabelClassifyAsync` on the `TextAnalyticsClient` by passing the documents as either an `IEnumerable<string>` parameter or an `IEnumerable<TextDocumentInput>` parameter. This returns a `ClassifyDocumentOperation`.
```

Articles should only end in a consonant when the first sound (or letter, depending on your chosen style) of the object is a vowel. I'd check other cases as well.

Reply:
Fixed!